packages/web-components/src/mixins/form-field-error.css

```diff
@@ -20,11 +21,6 @@
     margin-top: 0;
 }
 
-:host([error]:not([error=''])) label,
-:host([error]:not([error=''])) span {
```

Comment:
I think the intent behind the `:host([error]:not([error=''])) span` was for the `<span class="required">(*Required)</span>`. I'm fine with this change since we decided that only the error message should be bold, but I just wanted to call this out so that the `(*Required)` not being bold anymore doesn't come as a surprise.

## Before

![A text input component with a required prop and an error message. The label, error message, and "Required" text are all bold.](https://user-images.githubusercontent.com/2008881/190688213-cc696e12-acd7-4c4b-a975-f6beea4f2b48.png)

## After

![A text input with a required prop and an error message. Only the error message is bold.](https://user-images.githubusercontent.com/2008881/190688269-d538063f-f4f2-406a-9731-1c9617aa5ed7.png)